include/quda.h

```diff
@@ -638,6 +638,9 @@ extern "C" {
     /** Maximum number of iterations for refreshing the null-space vectors */
     int setup_maxiter_refresh[QUDA_MAX_MG_LEVEL];
 
+    /** Maximum number of iterations for inverse iteration polishing of null-space vectors */
+    int setup_maxiter_inverse_iterations_polish[QUDA_MAX_MG_LEVEL];
```

Comment:
For interface self-consistency, this should be _refinement_ rather than _polish_.

Reply:
Will do.

Reply:
https://github.com/lattice/quda/commit/5ac5288cf79843ba7c64c7cd48859130c48fc6e6